diff -Naur sage-old/build/pkgs/python2/spkg-install sage-new/build/pkgs/python2/spkg-install
--- sage-old/build/pkgs/python2/spkg-install	2017-07-12 18:23:32.367080391 +0000
+++ sage-new/build/pkgs/python2/spkg-install	2017-07-12 18:25:48.901847034 +0000
@@ -22,6 +22,9 @@
 
 cd src
 
+LDFLAGS="-lcrypt $LDFLAGS"
+export LDFLAGS
+
 if [ "$SAGE_DEBUG" = "yes" ]; then
     echo "Building Python with pydebug"
     PYTHON_CONFIGURE="$PYTHON_CONFIGURE --with-pydebug"
