diff -Naur old/build/pkgs/singular/spkg-install new/build/pkgs/singular/spkg-install
--- old/build/pkgs/singular/spkg-install	2017-07-11 21:05:49.036186347 +0000
+++ new/build/pkgs/singular/spkg-install	2017-07-11 21:09:03.325151372 +0000
@@ -142,10 +142,15 @@
     find singular -name '*.h' -exec sed -i.bak -f "$SRC/fix_includes.sed" {} \;
 }
 
+nix_nuke_bin_rm()
+{
+    find . -exec sed -e 's@/bin/rm@rm@g' -i '{}' ';'
+}
+
 
 # Actually run all the functions defined above
-for i in remove_old_version config \
+for i in nix_nuke_bin_rm remove_old_version config \
   build_singular fix_includes ; do
     echo "############### Singular stage $i ###############"
-    cd "$SRC" && $i
+    cd "$SRC" && pwd && $i
 done
